### DISCUSSION TITLE
Peak Height Lag

### DISCUSSION THREAD

**Discussion by martynw2000** [Chia-Network/chia-blockchain#3526] [Support]
Just a quick question - hoping someone has the answer.

I have three Windows PCs plotting, and while two of them seem to be perfectly in sync, one of them seems to be running about 50 blocks behind when I look at the peak height. The peak time is also off by a few minutes. The system time on all the PCs are in sync, and the status says "synced".

Should I restart the Chia app? I'm currently parallel plotting at 20% so I don't want to restart unless I have to. If it's not in perfect sync, does that mean that I will never win any challenges on this PC until it's corrected?

**Comment by richardhu2003** [comment]
Status shows that my peak height is always lagging behind the current network height, currently about 4000 behind. Can someone tell me how to ensure full sync at the same height?

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
